### PR TITLE
#162948874 Social Login

### PR DESCRIPTION
**Pull request by Deekerubo**
### What does this pull request do?
The pull request enables a user to login with their preferred social account as an alternative.

### Description of the tasks to be completed?
* The user is presented with buttons that enable them to log in with their social accounts.
* The following are the available social accounts
  `- GOOGLE`
  `- FACEBOOK`
  `- TWITTER`

### How should this be manually tested?
* clone the repo `https://github.com/andela/ah-alpha-frontend.git`
* change to the cloned directory `cd ah-alpha-frontend`
* checkout to this branch `git checkout ft-social-auth-162948874`
* run `npm install` to install the application.
* run `npm start`
* visit `https://ah-alpha.herokuapp.com/api/v1/social_auth/` to view the application on the browser.

### What are the relevant Pivotal Tracker Stories
[#162948874](https://www.pivotaltracker.com/story/show/162948874)

screenshot 2019-02-19 at 11 38 01

### CODE REVIEW

**Review by danuluma — APPROVED** [review]
LGTM

**Review by Ruiru11 — APPROVED** [review]
Good work

**Review by tesh254 — APPROVED** [review]
The Pull request is fine. You have removed unnecessary packages from the package.json

**Review by jamesbeamie — APPROVED** [review]
Looks ok.

File: src/actions/loginActions.js~HEAD [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import axios from "axios";
```
Comment (Betty-Kebenei) [inline src/actions/loginActions.js~HEAD]: Why do we have this file?

File: src/components/authentication/__test__/__snapshots__/Login.snapshot.test.js.snap [hunk 2]
```diff
@@ -0,0 +1,39 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (Betty-Kebenei) [inline src/components/authentication/__test__/__snapshots__/Login.snapshot.test.js.snap]: Kindly delete this snapshot

**Pull request merged**